benchmark/cajita/CMakeLists.txt

```diff
@@ -28,6 +31,9 @@ if(Kokkos_ENABLE_SERIAL OR Kokkos_ENABLE_CUDA OR Kokkos_ENABLE_OPENMP OR Kokkos_
     target_link_libraries(SparsePartitionerPerformance gtest)
     add_test(NAME Cajita_SparsePartitionerPerformance COMMAND ${NONMPI_PRECOMMAND} SparsePartitionerPerformance sparsepartitioner_output.txt)
 
+    target_link_libraries(InterpolationPerformance gtest)
```

Comment:
As Damien pointed out, this is unnecessary

Reply:
Removed.